src/config.js

```diff
@@ -1,6 +1,11 @@
 const convict = require('convict');
 
 const config = convict({
+  logLevel: {
+    doc: 'The logging level to set',
+    default: 'ALL',
```

Comment:
Let's default to warn, we don't need info and debug by default.

Reply:
ok sure. we can set WARN here but maybe we can try setting `INFO` on the first go and see how the logs look like?

most of our logs are info and as far as I can tell they don't log objects, so I'm hoping the logs would be ok.

Reply:
done